HW02/Alexey_Zalyotov/alexey_zalyotov_hw_02_t_03.rb

```diff
@@ -0,0 +1,34 @@
+require 'date'
+
+def events_timestamps
+  # logs information from file to array
+  logs = File.readlines("#{__dir__}/fixtures/data_hw_02_t_03.txt", chomp: true)
+  # find lines with "Calling core with action" substr and parse their timestamps
+  logs.select { |line| line =~ /.*Calling core.*/ }.map do |substr|
+    substr[/\S+\s\S+/]
```

Comment:
Can you move this regexp to constant? 🤔 It will help us avoid code duplication and would be more readable with a name 
```suggestion
    substr[DATE_TIME]
```

Reply:
Done